treeherder/services/pulse/consumers.py

```diff
@@ -162,6 +163,36 @@ def on_message(self, body, message):
         )
         message.ack()
 
+class Pulses(PulseConsumer):
```

Comment:
@armenzg our end goal could not have been possible with just an if-else.
In the ```pulse_listener``` we pass the repsective class of the ```env.json```
This was good when we had two different files, they used to pass two different classes. But now that we need a single push to have pulses for both, we need  a class which consists of both the bindings as well as different async function execution according to the exchange message.